public/index.html

```diff
@@ -28,7 +28,7 @@
     <meta property="og:title" content="EDLIZ  | Essential Drugs and Medicines List  for Zimbabwe" />
     <meta property="og:description" content="This 7th essential medicines list and standard treatment guidelines for the most common health conditions in Zimbabwe has been endorsed by the National Medicine & Therapeutics Policy Advisory Committee [NMTPAC]." />
     <meta property="og:image" content="https://edliz.azurewebsites.net/img/pills.jpg" /> 
-    <link href="https://fonts.googleapis.com/css?family=IBM+Plex+Serif|Muli:300,400,600" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=IBM+Plex+Serif|Muli:600" rel="stylesheet">
```

Comment:
Is there a reason why you decided to remove those font weights, are the fonts still working as expected after removing them?

Reply:
Yes, if you compare this PR to the live version of the source you will see that there was no place were the other font-weights were being used. So I thought it just reduces performance... unless I missed that.